src/pages/_app.tsx

```diff
@@ -23,6 +24,7 @@ import { SnackbarProvider } from 'hooks/SnackbarContext';
 import theme from '../theme';
 import { UserContext } from '../hooks';
 
+dayjs.extend(relativeTime);
```

Comment:
I don't think this is necessary if we consistently use `<FormattedRelativeTime>` in `JourneyInstancesDataTable` the same way it's used elsewhere.

Reply:
yup